daemon/cmd/local_node_sync.go

```diff
@@ -184,8 +184,11 @@ func (ini *localNodeSynchronizer) initFromK8s(ctx context.Context, node *node.Lo
 				node.SetCiliumInternalIP(net.ParseIP(addr.IP))
 			}
 		}
+
+		node.IPv4HealthIP = net.ParseIP(k8sCiliumNode.Spec.HealthAddressing.IPv4)
+		node.IPv6HealthIP = net.ParseIP(k8sCiliumNode.Spec.HealthAddressing.IPv6)
```

Comment:
There's one additional case to be handled here, namely if one of the IP families was enabled and got then disabled. In other words, we should set the IP only if the corresponding family is actually enabled.

Reply:
Done